src/packer.pkr.hcl

```diff
@@ -1,3 +1,18 @@
+variable "winrm_password" {
+  # Default is set due to a limitation. For more information
+  # see https://github.com/cisagov/windows-server-packer/issues/21.
```

Comment:
@dav3r @jsf9k Does this look ok to you as an updated comment? I wanted to be explicit about what's going on in line with my comment in the mentioned issue.

```suggestion
# This variable is configured as an optional variable because the
# cisagov/pre-commit-packer hook does not support passing variables to
# `packer validate`. Once this limitation is removed this should be changed to
# a required variable. Please see the following for more information:
# https://github.com/cisagov/windows-server-packer/issues/21
variable "winrm_password" {
```

Reply:
LGTM

Reply:
Looks great - thanks!